README.md

```diff
@@ -136,6 +137,15 @@ scheme://authority/package-name?version=x.x.x
 i.e. `ercxxx://packages.zeppelinos.eth/owned?version=1.0.0`
 
 
+## URI Backend Class
+* URI Backend Class defaults to `ethpm.backends.ipfs.IFPSGatewayBackend`
+* URI Backend Class is configurable via the environment variable `URI_BACKEND_CLASS` set to a dotted module path
```

Comment:
I'm of the opinion that these should be namespaced to prevent collisions (however unlikely).  (e.g. `ETHPM_URI_BACKEND_CLASS`)